nautobot/extras/api/nested_serializers.py

```diff
@@ -222,7 +230,19 @@ class Meta:
         fields = ["id", "url", "name", "slug"]
 
 
-class NestedScheduledJobSerializer(BaseModelSerializer):
+class NestedScheduledJobSerializer(WritableNestedSerializer):
+    url = serializers.HyperlinkedIdentityField(view_name="extras-api:scheduledjob-detail")
+    name = serializers.CharField(max_length=255, required=False)
+    start_time = serializers.DateTimeField(format=None, required=False)
+
+    class Meta:
+        model = models.ScheduledJob
+        fields = ["url", "name", "start_time", "interval", "crontab"]
+
+
+class NestedScheduledJobCreationSerializer(BaseModelSerializer):
+    """Unlike WritableNestedSerializer, this allows for creation of a ScheduledJob, not just finding an existing one."""
```

Comment:
```suggestion
    """
    Nested serializer specifically for use with `JobInputSerializer.schedule`.

    We don't use `WritableNestedSerializer` here because this is not used to look up
    an existing `ScheduledJob`, but instead used to specify parameters for creating one.
    """
```